SocialFilter/sections/specific.txt

```diff
@@ -55,11 +55,12 @@ note.calbee.jp,note.linemusic.jp,diehardtales.com,note.com,note.kagg.jp,note.jp,
 !
 lycoris-recoil.com##.l-header_share
 lycoris-recoil.com##.p-timeline
+sundaypost.com##iframe[id^="twitter-widget"]
```

Comment:
Where do you need this rule?

Reply:
For element on my screenshot

Reply:
Could you show me a screenshot of the element code? It's already covered by `##.twitter-timeline`.